src/trials/costBenefits.js

```diff
@@ -114,11 +120,46 @@ const costBenefits = (duration, blockSettings, opts, trialDetails) => {
         );
       };
 
-      trialDetails.probability = probability;
-      trialDetails.effort = effort;
-      trialDetails.high_effort = high_effort;
-      trialDetails.value = value;
-      trialDetails.subtrial_type = "cost_benefits";
+      /*canvasDraw();
+      function after_response(info) {
+        //clearInterval(timer);
+        jsPsych.pluginAPI.cancelKeyboardResponse(keyboardListener);
+        if (info.key === keys["Q"]) {
+          // 1 key
+          var returnObj = {
+           key: info.key,
+           effort: effort[0],
+           value: value[0],
+           high_effort: high_effort[0],
+           get_reward: get_reward[0],
+           subtrial_type: "choice",
+          };
+          done(returnObj);
+        } else if (info.key === keys["P"]) {
+          // 0 key
+          returnObj = {
+           key: info.key,
+           effort: effort[1],
+           value: value[1],
+           high_effort: high_effort[1],
+           get_reward: get_reward[1],
+          };
+          done(returnObj);
+        }
+      }
+        trialDetails.probability = probability;
+        trialDetails.effort = effort;
+        trialDetails.high_effort = high_effort;
+        trialDetails.value = value;
+        trialDetails.subtrial_type = "cost_benefits";
+
+      var keyboardListener = jsPsych.pluginAPI.getKeyboardResponse({
+        callback_function: after_response,
+        valid_responses: valid_keys,
+        rt_method: "date",
+        persist: true,
+        allow_held_key: false,
+      });*/
 
       canvasDraw();
```

Comment:
this whole chunk of code (~123-162) is commented out, yeah? do you need it for later? if not, my preference would be for this to be deleted as it'll clutter things up. we have github version control if you need this code again. again, not a big deal, just enforcing a style :-)

Reply:
I've since deleted it as Michael and I are working on it. also, how can I make these changes on the pull request without submitting a new PR? Haven't found a good answer online

Reply:
didn't you ask me this before, and i told you to push to this PR, which you were able to do the other day?